java/src/main/native/src/ColumnViewJni.cpp

```diff
@@ -1786,6 +1786,10 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnView_bitwiseMergeAndSetValidit
       default: JNI_THROW_NEW(env, cudf::jni::ILLEGAL_ARG_CLASS, "Unsupported merge operation", 0);
     }
 
+    if (cudf::may_have_nonempty_nulls(*copy)) {
```

Comment:
`may_have_nonempty_nulls` is more conservative and still causes performance regression. I've tried it and failed. Please use `cudf::has_nonempty_nulls`.

Reply:
Updated. Thank you!